README.md

```diff
@@ -32,6 +32,7 @@ for this step).
 - `--exclude <path>`: Exclude a subpath from root. Use multiple times to exclude multiple paths. Tags (imports/scripts/etc) that reference an excluded path are left in-place, meaning the resources are not inlined. ex: `--exclude=elements/x-foo.html --exclude=elements/x-bar.html`
 - `--inline-scripts`: Inline external scripts.
 - `--inline-css`: Inline external stylesheets.
+- `--redirect <prefix>|<path>`: Takes an argument in the form of `<prefix>|<path>` where `<prefix>` is an arbitrary url prefix, possibly including a protocol, hostname, and/or path prefix and `<path>` is a local filesystem path to route requests to.  Multiple redirects may be specified; the earliest ones have the highest priority.
```

Comment:
Not sure I understand what this is for after reading this. I guess it's not clear what a redirect means in the context of a bundler? Some way to clarify, maybe an example?